src/main/java/cz/xtf/radanalytics/util/configuration/RadanalyticsConfiguration.java

```diff
@@ -22,6 +22,7 @@
 	private static final String TEMPLATE_OSHINKO_PYTHON_SPARK_JSON="rad.oshinko.python.spark.template.json";
 	private static final String TEMPLATE_OSHINKO_JAVA_SPARK_JSON="rad.oshinko.java.spark.template.json";
 	private static final String TEMPLATE_OSHINKO_SCALA_SPARK_JSON="rad.oshinko.scala.spark.template.json";
+	private static final String OPENSTACK_INSTANCE_HOSTNAME_SUFFIX = "xtf.config.openstack.instance.hostname.suffix";
```

Comment:
Please rename this to `rad.openshift.instance.hostname.suffix` and the relevant variable as well

Reply:
done